src/trigger.py

```diff
@@ -50,8 +50,24 @@ def _run_trigger(self, build_config, force, timeout, trees):
             "owner": self._current_user['username'],
             "submitter": "service:pipeline"
         })
+        incomplete_node_count = self._api.node.count({
+            "kind": "checkout",
+            "data.kernel_revision.commit": head_commit,
+            "owner": self._current_user['username'],
+            "submitter": "service:pipeline",
```

Comment:
We can have a common `dict` with the below information and use it for calculating both `node_count` and `incomplete_node_count` (extend it with `result` field in this case).
```suggestion
       {
            "kind": "checkout",
            "data.kernel_revision.commit": head_commit,
            "owner": self._current_user['username'],
            "submitter": "service:pipeline",
```

Reply:
done